helm/mds/templates/deployment.yaml

```diff
@@ -129,3 +138,14 @@ data:
   postgresql-password: {{ .Values.postgresql.password | b64enc }}
   {{- end }}
 type: Opaque
+---
+{{- if false }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: stan-creds
+  namespace: {{ $.Release.Namespace }}
+data:
+  sys.creds: {{ $.Values.stanCreds | b64enc }}
+type: Opaque
+{{- end }}
```

Comment:
Should remove, unnecessary for vanilla nats, right?

Reply:
i'd like to keep this, because it is required for surveyor